ftd/image.ftd

```diff
@@ -262,14 +262,13 @@ dark: https://fastn.com/-/fastn.com/images/cs/show-cs-1-dark.jpg
 
 -- ds.h2: `fetch priority`
 
-Type: `optional` `string`
+Type: `optional` [`ftd.fetch-priority`](ftd/built-in-types/#ftd-fetch-priority)
 
 Required: False
 
-The `fetchPriority` property allows you to signal high or low priority image fetches.
-This can be useful when applied to <img> elements to signal images that are "important" to the user experience early in the loading process.
+The fetch-priority property allows you to signal high or low priority image fetches.This can be useful when applied to <img> elements to signal images that are "important" to the user experience early in the loading process.
```

Comment:
The line is too long, wrap lines at around 80 or so characters. Also use `fetch-priority` (backtick) for technical stuff, eg `<img>` is technical stuff and should be backtick quoted.